integration/containerd/confidential/lib.sh

```diff
@@ -180,44 +180,6 @@ switch_image_service_offload() {
 	esac
 }
 
-# Connect to the Pod's console (if it exists) to log the message into the file.
-#
-# Parameters:
-#       $1 - the pod name.
-#	$2 - file path.
-#
-# Return:
-#	the PID of the logger process. If the pod hasn't console then it
-#	will return empty.
-#
-# Note: use the enable_full_debug() method to ensure the VM's console
-#	will be configured.
-#
-# TODO: does it work with other hypervisors like clh?
-#
-crictl_record_cc_pod_console() {
```

Comment:
I was really proud of writing that function. So long dear code! :)

Reply:
It will always be with us in git commit history - until we squash everything when merging to `main`!!!